doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/getting-started.rst

```diff
@@ -213,38 +213,38 @@ After defining our configuration, we will use the Ray cluster launcher to start
 
     $ ray up -y config.yaml
 
-Run the application in the cloud
---------------------------------
+Running applications on a Ray Cluster
+-------------------------------------
 
-We are now ready to execute the application in across multiple machines on our Ray cloud cluster.
+We are now ready to execute an application on our Ray Cluster.
 ``ray.init()`` will now automatically connect to the newly created cluster.
 
-Next, run the following command:
+As a quick example, we can now execute a Python command on the Ray Cluster that connects to Ray and exits:
```

Comment:
micro-nit:
Considering removing "can now"
`As a quick example, we execute a Python...`